### PR TITLE
Fix QGIS plugin: no module grass

### PR DESCRIPTION
**Pull request by landam**
This PR solves QGIS plugin failure:

```py
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/qgis/utils.py", line 423, in _startPlugin
    plugins[packageName] = package.classFactory(iface)
  File "/home/martin/git/storm-fsv-cvut/smoderp2d/bin/qgis/smoderp2d-plugin/__init__.py", line 35, in classFactory
    from .smoderp_2D import Smoderp2D
  File "/usr/lib/python3/dist-packages/qgis/utils.py", line 888, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "/home/martin/git/storm-fsv-cvut/smoderp2d/bin/qgis/smoderp2d-plugin/smoderp_2D.py", line 31, in 
    from .smoderp_2D_dockwidget import Smoderp2DDockWidget
  File "/usr/lib/python3/dist-packages/qgis/utils.py", line 888, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "/home/martin/git/storm-fsv-cvut/smoderp2d/bin/qgis/smoderp2d-plugin/smoderp_2D_dockwidget.py", line 53, in 
    from .connect_grass import find_grass_bin
  File "/usr/lib/python3/dist-packages/qgis/utils.py", line 888, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "/home/martin/git/storm-fsv-cvut/smoderp2d/bin/qgis/smoderp2d-plugin/connect_grass.py", line 5, in 
    from smoderp2d.providers.grass import Popen
  File "/usr/lib/python3/dist-packages/qgis/utils.py", line 888, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "/home/martin/git/storm-fsv-cvut/smoderp2d/bin/qgis/smoderp2d-plugin/../../../smoderp2d/providers/grass/__init__.py", line 9, in 
    from smoderp2d.providers.grass.logger import GrassGisLogHandler
  File "/usr/lib/python3/dist-packages/qgis/utils.py", line 888, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "/home/martin/git/storm-fsv-cvut/smoderp2d/bin/qgis/smoderp2d-plugin/../../../smoderp2d/providers/grass/logger.py", line 4, in 
    from grass.script.core import info, error, warning, debug, percent
  File "/usr/lib/python3/dist-packages/qgis/utils.py", line 888, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
ModuleNotFoundError: No module named 'grass'
```

### CODE REVIEW

File: smoderp2d/runners/arcgis.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+from smoderp2d.runners.base import Runner
+
+class ArcGisRunner(Runner):
+    """TODO."""
+
+    def __init__(self):
+        os.environ['ESRIACTIVEINSTALLATION'] = '1'
+        super(ArcGisRunner, self).__init__()
+
+    def _get_provider(self):
```
Comment (pesekon2) [inline smoderp2d/runners/arcgis.py:11]: @landam: Don't you want to add a docstring here?

File: smoderp2d/runners/grass.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+from smoderp2d.runners.base import Runner
+                    
+class GrassGisRunner(Runner):
+    """TODO."""
+    def _get_provider(self):
```
Comment (pesekon2) [inline smoderp2d/runners/grass.py:5]: @landam: Don't you want to add a docstring here?

File: smoderp2d/runners/qgis.py [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+        # calling gsetup.init() is not enough for PyGRASS
+        Mapset('PERMANENT', location, gisdb).current()
+
+        # test GRASS env varible
+        if not os.getenv('GISRC'):
+            raise SmoderpError('GRASS not found.')
+
+        super().__init__()
+
+    def _get_provider(self):
```
Comment (pesekon2) [inline smoderp2d/runners/qgis.py:68]: @landam: Don't you want to add a docstring here?

**Review by pesekon2 — APPROVED** [review]

File: smoderp2d/runners/arcgis.py (hunk 1 above)
Comment (landam) [reply smoderp2d/runners/arcgis.py:10]: Solved by 9accb4c8

**Pull request merged**